openedx/core/djangoapps/programs/utils.py

```diff
@@ -47,11 +47,16 @@
 def get_program_marketing_url(programs_config, mobile_only=False):
     """Build a URL used to link to programs on the marketing site."""
     if mobile_only:
-        marketing_url = 'edxapp://course?programs'
-    else:
-        marketing_url = urljoin(settings.MKTG_URLS.get('ROOT'), programs_config.marketing_path).rstrip('/')
+        return 'edxapp://course?programs'
 
-    return marketing_url
+    # This is using LOGOUT_REDIRECT_URL as it already contains the
+    # desired value for the marketing site.
+    marketing_url = configuration_helpers.get_value('LOGOUT_REDIRECT_URL', None)
```

Comment:
Why don't you just add another site setting?. What happen if then we are asked to redirect to a different page after logout?. @Squirrel18

Reply:
I wanted to avoid two or more different settings with the same value, but I changed it. @Jacatove